.travis.yml

```diff
@@ -14,6 +14,15 @@ before_script:
 script:
   - gulp
 
+deploy:
+  provider: npm
+  email: "john@radify.io"
```

Comment:
@nateabele If we use `shipping@radify.io`, that email will also need to have a Github account, an NPM account, and an NPM API key. Regarding, `npm publish`, no one person can become a single point of failure so long as multiple contributors are listed in `package.json` and on the `npmjs.com` website. Regarding automated Travis publication, my credentials _could_ become a point of failure if I changed by NPM API key, but even that could be fixed with a simple change of the `.travis.yml`. Should we go ahead & switch to `shipping@radify.io`? Additionally, I'm looking into [NPM Organizations](https://docs.npmjs.com/getting-started/working-with-orgs), which seems to be a new thing:tm:, and might solve problems like this for us.